test/PrintAsObjC/public-symbols-only.swift

```diff
@@ -0,0 +1,14 @@
+// REQUIRES: objc_interop
+
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend -emit-module -o %t %s -module-name PublicSymbols
+// RUN: %target-swift-frontend -parse-as-library %t/PublicSymbols.swiftmodule -typecheck -emit-objc-header-path %t/result.h -import-objc-header %S/../Inputs/empty.h -objc-header-is-public
```

Comment:
Just checking: Does this test fail if you omit `-objc-header-is-public`? I'm not sure if this module falls into the "lousy approximation".

Reply:
it should. But I think the lousy approximation should go away and the build system should dictate what are printed in these headers.